cmd/wallet-sdk-gomobile/credential/inquirer.go

```diff
@@ -28,6 +28,7 @@ import (
 // Inquirer implements querying credentials using presentation definition.
 type Inquirer struct {
 	goAPICredentialQuery *credentialquery.Instance
+	goDIDResolver        goapi.DIDResolver
```

Comment:
```suggestion
	goAPIDIDResolver        goapi.DIDResolver
```

Just for consistency with the other field names